### DISCUSSION TITLE
How to configure new networks/nodes via configuration files

### DISCUSSION THREAD

**Discussion by DavideM78** [gravitl/netmaker#2269] [General]
Hi everybody and thanks to anyone will answer to my very simple question.
It is possible to configure Netmaker, I mean add, configure, modify and delete a network and/or a node via configuration files instead of doing it on the UI? I didn´t find anything on the docs so I assume it is not possible, but I would like to be sure.
Best regards, 
Davide

**Comment by mattkasun** [comment]
You can use nmctl instead of the UI.  You could write a script that reads a config file and calls nmctl
